### PR TITLE
Replace rospublic with rosmaster

### LINKED ISSUE #1840: Replace rospublic in astroquery/heasarc/tests/test_heasarc_remote.py
`astroquery/heasarc/tests/test_heasarc_remote.py` has multiple references to `rospublic`. This is no longer a valid HEASARC table after having been deprecated 16 years ago, so all these tests will fail. Replace with `rosmaster` or some other table. The number of rows in the results may need to updated to reflect the query results of the new test table.

### PR DESCRIPTION
**Pull request by daisycodes**
Closes #1840

### CODE REVIEW

File: CHANGES.rst
Comment (bsipocz) [inline CHANGES.rst:732]: No need for a changelog entry.  We don't add entries for changes that made only to the documentation or tests, or some internal refactors that are invisible to users (rule of thumb is whether it's something that affects users or not). 
I'll remove your second commit before merging, so no need to any follow-ups here.

**Review by bsipocz — APPROVED** [review]

File: CHANGES.rst
Comment (bsipocz) [reply CHANGES.rst]: (when a maintainer adds the `no-changelog-entry-needed` label, the check status will get fixed below)

**Comment by bsipocz** [comment]
Thank you @daisycodes for your first PR to astroquery, and welcome!

**Comment by daisycodes** [comment]
Thanks @bsipocz. I'm super excited about my first open source contribution.

> No need for a changelog entry. We don't add entries for changes that made only to the documentation or tests, or some internal refactors that are invisible to users (rule of thumb is whether it's something that affects users or not).
> I'll remove your second commit before merging, so no need to any follow-ups here.

Thanks for this. I added that when it failed the test on GitHub.

**Pull request merged**
